community/contributors/contributors-name-mapping.json

```
@@ -54,6 +55,7 @@
   "kariem": "Kariem Hussein",
   "kevinpollet": "Kevin Pollet",
   "khalidq": "Khalid",
+  "koentsje": "Koen Aers",
   "lalmeras": "Laurent",
   "lordofthejars": "Alex Soto",
   "lukasz-antoniak": "Lukasz Antoniak",
```

can we add 
```
"lucamolteni":  "Luca Molteni", 
```

as well?

---

Sure :) In fact I'll submit another PR with all these unrelated changes.

---

#232

---

I'm not there as well 🙈 

```suggestion
  "lukasz-antoniak": "Lukasz Antoniak",
    "mbladel": "Marco Belladelli",
```

---

Added to #232